### ISSUE TITLE
Is it possible to set the line length limit?

### ISSUE THREAD

**Issue by jasjuang** [20Tauri/DoxyDoxygen#22]
Would it be possible to limit the line length to a fix number (for example 80) when alt + Q is used? Would be great to have this feature. 

EDIT: one observation is the line length is always a little over 80, I further realize if I unindent the comment block so it starts at the very left it will be never exceed 80. So it seems to be limiting the line length to 80 but wrongly not counting the indents. Might be a bug. Would appreciate if you can fix it!

**Comment by JohannesMP** [comment]
Not sure if this is what you're asking, but you should look at this in the settings file:

```
// "prefered_line_length" (the value used on wrap command)  is based on
// SublimeText "rulers" parameter (default: 80 if empty)
```

Have you tried setting `prefered_line_length` to your desired value?

**Comment by jasjuang** [comment]
This is exactly it but it is not behaving as expected. My "rulers" is set to 80. I tried setting "prefered_line_length" directly to 80 and the same problem is still there. 

EDIT: For example assuming the function below is in a class header with 2 space indention, it exceeds 80 and didn't break automatically with alt + Q

```
  /**
   * @brief      Get the foreground map, white is foreground, black is background
   */
  void getForeGround();
```

**Comment by 20Tauri** [comment]
There's no specific plugin parameter called "preferred_line_length", I only use "rulers" in "users" settings instead.

And currently, I don't take care of both
- the documentation block indentation
- the documentation block style

It's in my todo list...

**Comment by jasjuang** [comment]
Thanks, look forward to see it fixed!

**Comment by 20Tauri** [comment]
0.20.0 has been release.
This version fix this issue for comment body (but, long "comment_styles" keep there fixed size)

**Comment by jasjuang** [comment]
Thanks for the update, but I tried the example with 2 space indention and alt + Q still would not break the line

```
  /**
   * @brief      Get the foreground map, white is foreground, black is background
   */
  void getForeGround();
```

**Comment by 20Tauri** [comment]
Making deeper test I've found a small bug with limit value...

Bug it's probably not your test case.... Have you restart SublimeText after update ?

If not, please restart SublimeText and check again... Update may cause problem.... It's really technical, but even if I've already fix some bugs in python ZipImporter cache, I still have problem on some configuration (hard to reproduce)... 0.20.0 is designed to prevent crash in those cases... For the story, on a previous version, I've received more than fifty crash report (even if this is written in manual and in integrated report agent) ;-(... Time consuming.

If problem persist after reboot please give me your settings.

**Comment by jasjuang** [comment]
I re-installed doxydoxygen (evolution) and re-started sublime and re-started my computer and the problem is still there.

The doxydoxygen setting is default, I have nothing in user settings.

For general sublime settings, my user setting is

{
    "color_scheme": "Packages/User/SublimeLinter/Mac Classic (SL).tmTheme",
    "font_size": 10,
    "ignored_packages":
    [
        "Vintage"
    ],
    "remember_open_files": true,
    "rulers":
    [
        80
    ],
    "word_wrap": false
}

Were you able to reproduce the problem?

**Comment by 20Tauri** [comment]
It's exactly to problem on ruler limit I've explain in may previous message.
Fixed in 0.20.1

**Comment by jasjuang** [comment]
Thank you so much it is working perfect now!
